runelite-api/src/main/java/net/runelite/api/ItemComposition.java

```diff
@@ -38,6 +38,13 @@
 	 */
 	String getName();
```

Comment:
Updated the javadoc. Still not entirely confidant with writing good docs but hopefully this is good enough.

Reply:
I think the content is fine, but the formatting isn't right as-is. Since Javadoc is just HTML, this ends up yielding the following formatting when generated:

> #### getName()
> java.lang.String getName()
>
> Gets the items name as it appears in game. On a members server, this is always the item's actual name. On a free server, this depends on [Client.setOverrideMembersItemNames(boolean)](): * If it has not been called, or called with false, members items will have the name "Members object" * If it has been called with true, members items will have their actual name with " (members)" appended eg. Twisted bow (members)
>
> **Returns:**
> the name of the item as it appears in game

Primarily, this just needs the list to have list markup be HTML rather than markdown, eg.

```
[ ... ] this depends on {@link Client#setOverrideMembersItemNames}:
<ul>
  <li>If it has not been called, or called with false, members items will have the name "Members object"</li>
  <li>If it has been called with true, members items will have their actual name with " (members)" appended eg. "Twisted bow (members)"</li>
</ul>

[ ... ]
```